src/js/components/List/__tests__/List-test.js

```diff
@@ -600,11 +600,11 @@ describe('List onOrder', () => {
     // beta's up arrow control should be active
     expect(container.firstChild).toMatchSnapshot();
     fireEvent.keyDown(getByText('alpha'), {
-      key: 'Enter',
-      keyCode: 13,
-      which: 13,
+      key: 'Space',
```

Comment:
Did we need to change this test? Seems like `Enter` is supported

Reply:
nice ☝️ thinking